# This expresses the inertial part of the xState, as well as its error
# covariance.

Header    header
float64[] state
float64[] cov
